dataplane/pkg/common/dataplane.go

```diff
@@ -32,6 +34,7 @@ import (
 	monitor_crossconnect "github.com/networkservicemesh/networkservicemesh/controlplane/pkg/monitor/crossconnect"
 	"github.com/networkservicemesh/networkservicemesh/dataplane/pkg/apis/dataplane"
 	"github.com/networkservicemesh/networkservicemesh/pkg/tools"
+	. "github.com/networkservicemesh/networkservicemesh/sdk/vppagent/dataplane"
```

Comment:
I believe the common part of the dataplane should not refer to any specific forwarding plane implementation.

Reply:
yep, you are right. This just for test on CI. PR has prefix work in progress

Reply:
Oh, great! 👍